cmd/osbuild-worker/jobimpl-osbuild.go

```diff
@@ -160,11 +162,21 @@ func (impl *OSBuildJobImpl) Run(job worker.Job) error {
 			}
 
 			/* TODO: communicate back the AMI */
-			_, err = a.Register(t.ImageName, options.Bucket, key, options.ShareWithAccounts, common.CurrentArch())
+			ami, err := a.Register(t.ImageName, options.Bucket, key, options.ShareWithAccounts, common.CurrentArch())
 			if err != nil {
 				r = append(r, err)
 				continue
 			}
+
+			var amiResult string = ""
+			if ami != nil {
```

Comment:
This should probably be treated as an error, don't you think? Probably the underlying library already does, but just in case, it might makes sense to error out here. Also feel free to leave as is, up to you!

Reply:
You're right, thanks for pointing it out!